fairscale/optim/oss.py

```diff
@@ -597,4 +593,4 @@ def _setup_flat_buffers(self) -> None:
                     else:
                         self.buckets[device][dst_rank] = bucket
                 else:
-                    self.buckets[device].append(torch.zeros(1, device=device))
+                    self.buckets[device].append(torch.zeros(0, device=device))
```

Comment:
an empty tensor is a thing, better take to catch empty shards

Reply:
maybe add this same line as a comment for better code readability?

Reply:
hmm ok, the comment was about the change actually but if that helps I can always write something. The if clause is if(no params) so I thought that was kind of clear (outside of this very narrow PR view)